catalog/installGit.sh

```diff
@@ -20,7 +20,7 @@ install "$CATALOG_HOME/github/webhook.js" \
     github/webhook \
     -a feed true \
     -a description 'Creates a webhook on github to be notified on selected changes' \
-    -a parameters '[ {"name":"username", "required":true, "bindTime":true}, {"name":"repository", "required":true, "bindTime":true}, {"name":"accessToken", "required":true, "bindTime":true},{"name":"events", "required":true} ]' \
+    -a parameters '[ {"name":"username", "required":true, "bindTime":true}, {"name":"repository", "required":true, "bindTime":true}, {"name":"accessToken", "required":true, "bindTime":true}, {"name":"endpoint", "required":false, "bindTime:true}, {"name":"events", "required":true} ]' \
```

Comment:
@lzbj I believe this is not quite what @rabbah was suggesting.  Please see e0a628ea6fc851c85a381f9618737012c7e482c3 for how to set and use the default value as a parameter on the package itself.